src/services/NotificationService.ts

```diff
@@ -36,12 +37,14 @@ class NotificationService {
     }
 
     if (emailType === EMAIL_TYPE.ATF) {
```

Comment:
Not sure if I am missing something, but wasn't this ticket to also to add some logic so that it fails gracefully in the event there is an issue when sending the e-mail?

As far as I can tell, the issue we were having before is that when the sendNotification method was being called it was throwing an error (due to an invalid email address in the test station emails list, 400 was being returned). 

The method is called twice - lines 44 and 48 of SendATFReport. The first call sends e-mails to those listed against the test-station, the second call sends to the tester. If an error is thrown in the first call, would that not mean the second call to send to the tester is missed?